### PR TITLE
Close #52. Add project info to readme

### LINKED ISSUE #52: Create Project Readme
Make it pretty.

### PR DESCRIPTION
**Pull request by ryanflach**
- Minor refactoring

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -9,8 +9,8 @@ class Item < ApplicationRecord
   def self.most_revenue(quantity)
     unscoped.
       select(
-        "items.*, "\
-        "SUM((invoice_items.unit_price::float / 100) "\
```
Comment (ryanflach) [inline app/models/item.rb:13]: Removed as this formatting is now handled in our JBuilder helpers.

File: app/models/item.rb [hunk 2]
```diff
@@ -22,10 +22,7 @@ def self.most_revenue(quantity)
 
   def self.top_items_by_number_sold(quantity)
     unscoped.
-      select(
-        "items.*, "\
-        "SUM(invoice_items.quantity) AS most_items"
-      ).
```
Comment (ryanflach) [inline app/models/item.rb:28]: Made this a single line as it was under the 80 char limit and was more readable.

**Comment by ryanflach** [comment]
@pwentz All tests still passing.

File: app/models/item.rb (hunk 1 above)
Comment (pwentz) [reply app/models/item.rb:13]: Agreed. Let's keep that formatting logic in one place.

**Comment by pwentz** [comment]
That `README.md` is outstanding. Great work 👌

**Pull request merged**